oak/client/policy_metadata.cc

```diff
@@ -19,18 +19,19 @@
 #include <map>
 #include <utility>
 
+#include "absl/strings/escaping.h"
 #include "oak/common/policy.h"
 
 namespace oak {
 
-PolicyMetadata::PolicyMetadata() {}
+PolicyMetadata::PolicyMetadata(const oak::policy::Labels& labels) {
```

Comment:
It might be nicer to write this as:
```c++
PolicyMetadata::PolicyMetadata(const oak::policy::Labels& labels) : serialized_policy_(SerializePolicy(labels)) {}
```
then `serialized_policy_` could be made `const` in the class definition.

Reply:
Done.